python/paddle/fluid/incubate/fleet/collective/__init__.py

```diff
@@ -69,6 +84,78 @@ def save_persistables(self, executor, dirname, main_program=None):
 fleet = Collective()
 
 
+class CollectiveOpBasedOptimizer(DistributedOptimizer):
```

Comment:
the similar names, CollectiveOpBasedOptmizer and CollectiveOptimizer, make me confused a bit, as CollectiveOptimizer is original, could we change CollectiveOpBasedOptmizer to some other? As u r the NAMING MASTER, do u have any ideas?

Reply:
I generally want to make CollectiveOpBasedOptmizer as a base class for all DistributedOptimizers that use collective op to do optimization. Maybe CollectiveOpOptimizerBase is better.